dbms/src/Encryption/RateLimiter.cpp

```diff
@@ -455,10 +435,9 @@ void IORateLimiter::updateReadLimiter(Int64 bg_bytes, Int64 fg_bytes)
 {
     LOG_FMT_INFO(log, "updateReadLimiter: bg_bytes {} fg_bytes {}", bg_bytes, fg_bytes);
     auto get_bg_read_io_statistic = [&]() {
-        return getCurrentIOInfo().bg_read_bytes;
+        return io_info.bg_read_bytes;
```

Comment:
`io_info` may be accessed by `ReadLimiter` and background thread concurrently.

Reply:
`io_info` will only write by `auto_tune_and_get_io_info_thread`, may it is unnecessary to add a lock?